auth0/k8s-pixy-auth
# feat: configurable callback port

By submitting a PR to this repository, you agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md). Please see the [contributing guidelines](https://github.com/auth0/.github/blob/master/CONTRIBUTING.md) for how to create and submit a high-quality PR for this repo.

### Description

Makes the callback port configurable.

### References

Fixes #27 

### Testing


- select a port (e.g. `1337`)
- make sure the redirect url is correctly configured in the IDP (for our example it must be `http://127.0.0.1:1337/callback`)
- pass the selected port to k8s-pixy-auth using the parameter `--port` 

### Checklist
